apps/src/templates/manageStudents/manageStudentsRedux.js

```diff
@@ -413,19 +432,22 @@ const updateStudentOnServer = (updatedStudentInfo, onComplete) => {
   });
 };
 
-// Make a post request to add a student.
-const addStudentOnServer = (updatedStudentInfo, sectionId, onComplete) => {
-  const studentToAdd = [{
-    editing: true,
-    name: updatedStudentInfo.name,
-    age: updatedStudentInfo.age,
-    gender: updatedStudentInfo.gender,
-  }];
+// Make a post request to add students.
+const addStudentOnServer = (updatedStudentsInfo, sectionId, onComplete) => {
+  const studentsToAdd = [];
+  for (let i = 0; i<updatedStudentsInfo.length; i++) {
+    studentsToAdd[i] = {
+      editing: true,
+      name: updatedStudentsInfo[i].name,
+      age: updatedStudentsInfo[i].age,
+      gender: updatedStudentsInfo[i].gender,
+    };
+  }
```

Comment:
This looks like a `map` operation.

Reply:
Oops missed this one - but will follow up.